### PR TITLE
a11y: fix dropdown up/down key press scrolling the page

### PR DESCRIPTION
**Pull request by eranshabi**
This fix unintended behavior where pressing up & down arrows scroll the outer page as well as selecting dropdown items

### CODE REVIEW

File: packages/wix-ui-core/src/components/popover/Popover.driver.ts [hunk 1]
```diff
@@ -17,5 +17,6 @@ export const popoverDriverFactory = ({element, eventTrigger}) => ({
     return {top, left, right, bottom};
   },
   inlineStyles: () => element.style,
-  getElementId: () => element.id
+  getElementId: () => element.id,
+  popoverKeyDown: (key, preventDefault?) => eventTrigger.keyDown(element.querySelector('[data-hook="popover-element"]'), preventDefault ? {key, preventDefault} : {key})
```
[the remark this answers is not in the document]
Comment (eranshabi) [reply packages/wix-ui-core/src/components/popover/Popover.driver.ts]: It is used, just search `popoverKeyDown` in `Dropdown.spec.tsx`.
The  button must be pressed inside the popover, or else I wouldn't be able to reproduce the bug in the test. In reality the popover is focused so every key press is triggered on it too.

[the remark this answers is not in the document]
Comment (shlomitc) [reply packages/wix-ui-core/src/components/popover/Popover.driver.ts]: it still sounds very strange that the Popover should get the event.
According to the popover API, the consumer provides an `Element` component. this component can be anything that is being referenced as the source of the popover.

why not trigger the event on such custom element?
it looks like an issue dropdown only and not part of the Popover

**Comment by eranshabi** [comment]
I added a storybook for dropdown, not sure how to see it?

File: packages/wix-ui-core/src/components/popover/Popover.driver.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (eranshabi) [reply packages/wix-ui-core/src/components/popover/Popover.driver.ts]: I tried and it doesn't work in tests, not sure way.
I can move the `popoverKeyDown` function to the dropdown.driver.
The keyDown callback is passed to `Popover` in the `Dropdown` component, so it's something the component knows about.
Will moving the function to dropdown.driver be OK @shlomitc ?

**Pull request merged**